webanno-brat/src/main/java/de/tudarmstadt/ukp/clarin/webanno/brat/resource/visualizer_ui.js

```diff
@@ -335,7 +335,7 @@ var VisualizerUI = (function($, window, undefined) {
                 }
 
             var displaySpanComment = function(
-                evt, target, spanId, spanType, mods, spanText, commentText, commentType, normalizations) {
+                evt, target, spanId, spanType, mods, spanText, hoverText, commentText, commentType, normalizations) {
```

Comment:
When making changes to the JS files, please always surround the changes with a prominent comment.
The changes for this issue in `visualizer_ui.js` don't exhibit these markers.

Reply:
sorry, forgot it, I added comments now (4803b4fcf9bcdcc3965415f1a67c3ec1e0fd5363)